atsign-foundation/at_wavi_app
# Fixes #148

**- What I did**
      I used the url_launcher package to open google maps. When you add a location to your page, It displays a map and a location title. In that map tile, I have added a new icon below the fullscreen icon. So it will open google maps or apple maps according to respective coordinates.

**- How I did it**
     I have used the url_launcher package and written a function named openMaps to google maps or apple maps.
It will open the app according to the device.

**- How to verify it**
     To verify it, click on the location icon in the map tile (below the fullscreen icon). It will open the map in the respective app.

**- Description for the changelog**
      Added feature to open a location inside google maps or Apple maps.
